README.md

```diff
@@ -14,8 +15,6 @@ Your Flutter code for handling BLE will not be aware that it doesn't run on the
 
 The simulation allows you to develop BLE applications on iOS simulator and/or Android emulator, and run automated tests.
 
-## Configuration and installation
```

Comment:
Shouldn't we leave this? Repository is called blemulator_flutter, the package is called blemulator. I think we should add a snippet how the pubspec.yml entry should look like

Reply:
I have checked the l all packages from the top list on pub.dev and all of them have only a badge that links to a pub.dev. But I have not a strong opinion so if you think it is valuable I could Add some installation instruction in the Readme

Reply:
We have discussed it, tl; dr; I will add a minimalistic section `Usage`